src/widgetastic_patternfly5/components/expandable_section.py

```diff
@@ -0,0 +1,44 @@
+from wait_for import wait_for_decorator
+from widgetastic.widget import View
+
+
+class ExpandableSection(View):
+    """Represents the Patternfly Expandable section widget.
+
+    Expands itself automatically when any child widget gets accessed, ensuring that the widget is
+    visible.
+
+    https://www.patternfly.org/components/expandable-section
+    """
+
+    ROOT = './/div[contains(@class, "-c-expandable-section")]'
+    BUTTON_LOCATOR = ".//button"
+
+    def is_expanded(self):
+        """Returns a boolean."""
+        if self.browser.get_attribute("aria-expanded", self.BUTTON_LOCATOR) == "true":
+            return True
+        else:
+            return False
+
+    @property
+    def is_displayed(self):
+        """Returns a boolean detailing if the section is displayed."""
+        return self.browser.is_displayed(self.ROOT)
+
+    def click(self):
+        """Clicks the expandable section button."""
+        return self.browser.click(self.BUTTON_LOCATOR)
+
+    def expand(self):
```

Comment:
Better to have opposite method as well ^^ with some better name like `shrink` or something other.